website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+
+In dbt Core, environment variables, or the [`env_var` function](/reference/dbt-jinja-functions/env_var), are defined in the `profiles.yml` file.
+In dbt Cloud, you can set [environment variables](/docs/build/environment-variables) in the dbt Cloud user interface (UI). Read [Set up environment variables](#set-environment-variables-in-dbt-cloud) for more info.
+
+### Environment variables in dbt Cloud
+  - dbt Cloud environment variables must be prefixed with `DBT_` (including `DBT_ENV_CUSTOM_ENV_` or `DBT_ENV_SECRET_`).
+  - If your dbt Core environment variables don’t follow this naming convention, perform a “find and replace” in your project to make sure all references to these environment variables contain the proper naming conventions.
+- dbt Cloud secures environment variables, offering additional measures for sensitive values, such as prefixing keys with `DBT_ENV_SECRET_` to obscure them in logs and the UI.
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Environment Variables/project-environment-view.png" title="Setting project level and environment level values"/>
+
+### dbt Cloud environment variables order of precedence
+Environment variables in dbt Cloud are managed with a clear order of precedence, allowing users to define values at four levels (highest to lowest order of precedence):
+   - The job level (job override) or in the IDE for an individual developer (personal override). _(Highest precedence)_
+   - The environment level, which can be overridden by the job level or personal override.
+   - A project-wide default value, which can be overridden by the environment level, job level, or personal override.
+   - The optional default argument supplied to the `env_var` Jinja function in code. _(Lowest precedence)_
+  
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Environment Variables/env-var-precdence.png" title="Environment variables order of precedence"/>
+
+### Set environment variables in dbt Cloud
+
+- To set these variables for an entire project or specific environments, navigate to **Deploy** > **Environments** > **Environment variables** tab.
+- To set these variables at the job level, navigate to **Deploy** > **Jobs** > **Select your job** > **Settings** > **Advanced settings**.
+- To set these variables at the personal override level, navigate to **Profile Settings** > **Credentials** > **Select your project** > **Environment variables**.
+
+## Orchestration setup
+
+This section outlines the considerations and methods to set up your dbt Cloud environments and jobs for orchestration. The following categories are covered in this section:
+
+- [dbt Cloud environments](/guides/core-to-cloud-1?step=8#dbt-cloud-environments-1)
+- [Initial setup steps](/guides/core-to-cloud-1?step=8#initial-set-up-steps-1)
+- [Additional configurations](/guides/core-to-cloud-1?step=8#additional-configurations)
+- [CI/CD setup](/guides/core-to-cloud-1?step=8#cicd-set-up)
```

Comment:
```suggestion
- [CI/CD setup](/docs/deploy/ci-jobs)
```

Reply:
the rest of the links link to the sections of the same page so changed it to the ci/cd section of the page, rather than the actual docs page. the docs page is already linked in that section 

![Screenshot 2024-03-14 at 12 03 01](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/3ca505b5-a9b3-4d1f-a311-e2c5bad96107)